interview-questions.md

```diff
@@ -7,52 +7,52 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What are props in React?
 
-  Your answer:
+  Your answer: 'props' in react stand for properties and is what connects the parent to the child. 
 
-  Researched answer:
+  Researched answer: Are tagged by HTML and are passed as arguments in react, it passes data from one component to another (parent child) and is contained in state. It allows users to interact with the information between parent and child components, it can only be passed from parent to child.
 
 
 
 2. What is a DOM event?
 
-  Your answer:
+  Your answer: Is a signal for when something's occured through browser.
 
-  Researched answer:
+  Researched answer:  when HTML tags are triggers through user interaction on the website. Also stores data that always users to interact with the website and it happens before the event has occured. 
 
 
 
 3. What is object-oriented programming? How is it different than functional programming?
 
-  Your answer:
+  Your answer: Functional programming stores and manipulates data by functions. Object-oriented stores and manipulates data through objects.
 
-  Researched answer:
+  Researched answer: Object oriented programming stores and operates in a collection of objects. Concepts of  OOP are abstraction: where the important parts of an object's properties and behaviors are  identified and viewed, these behaviors are also what diffrenciate it from other objects. Encapsulation wraps up data in a class. Inheritance, where one class can use features listed from another class. Polymorphism can differentiate between data listed under the same name. Functional programming the data is fixed, its main concept is abstarction of data. It's composed of functions only
 
 
 
 4. What is the difference between a Float and an Integer in Ruby?
 
-  Your answer:
-
-  Researched answer:
+  Your answer: Floats are not whole numbers, they're decimals. Integers are whole numbers
 
+  Researched answer: Integers are whoole numbers without decimal points, you can perform arithamtic  and the return will always be an integer. There's built-in methods that can be used on integers for example: .even
+  and  .odd. Floats are frational numbers (decimal values), built-in methods can be used on floats to convert them to whole numbers .ceil rounds a float up and .floor rounds a float down.
 
 
 5. Ruby has an implicit return. What does that mean?
 
-  Your answer:
+  Your answer: Returns the value of the code put in automatically.
 
-  Researched answer:
+  Researched answer: Does not really need a return key , it automatically returns the last value evaluated.
```

Comment:
You did a great job defining the concepts in both your answers and researched answers.  I would like to see your answers be expanded upon a little further as this is supposed to mimick how you would respond to an interviewer in an interview.